woocommerce/woocommerce
# Fix bug introduced in ef57248

### All Submissions:

I matched the style of the file.

### Changes proposed in this Pull Request:

This fixes a bug introduced in ef57248 wherein Numbers are no longer supported by the `$.payment.validateCardExpiry()` and `$.payment.validateCardCVC()` functions. This is important because `$.payment.cardExpiryVal()` returns Numbers. If you then pass the results of that function into the validation function, it will fail incorrectly.

This commit uses the technique used by the last version of the `$.trim()` function. This will fix all backward-compatibility breaks.

See: https://github.com/jquery/jquery/blob/3.6.0/src/deprecated.js#L82-L86

### How to test the changes in this Pull Request:

1. In console, run and verify output is `true`:
2. `jQuery.payment.validateCardExpiry(1, 2022)`
3. `jQuery.payment.validateCardCVC(123)`

### Other information:
